styles.css

```diff
@@ -637,8 +657,8 @@ body button:hover {
 @media (min-width: 768px) {
   .contact {
     padding-bottom: 1.5em;
-    background-image: url('/img/contactIzq-background.svg'), url('/img/contact-center-desktop.svg'), url('/img/contact-der-desktop.svg');
-    background-position: bottom left, 20em 15em, bottom right;
+    background-image: url('img/contactIzq-background.svg'), url('img/contact-center-desktop.svg'), url('img/contact-der-desktop.svg');
+    background-position: bottom left, 20em 28em, bottom right;
     background-repeat: no-repeat;
   }
```

Comment:
Great job @taldr27 🤝

- [ ] [Optional] Kindly align your elements properly on your design so as to replicate the Figma design.
- [ ] [Optional]You may also kindly make the text color the same as the one in the Figma template. This will make your design look the same as the one provided in the Figma template.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/186823993-84153229-c10f-4f2e-b009-60090da51848.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/186824147-bfc382f5-af01-4f8e-af4e-70f70569ca52.PNG)